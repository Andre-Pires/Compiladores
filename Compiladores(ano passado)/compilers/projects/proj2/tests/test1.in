float f(int a){
  int x;
  float y;
  x=12;
  y=23;
  x=3.78;
  return 0;
}
