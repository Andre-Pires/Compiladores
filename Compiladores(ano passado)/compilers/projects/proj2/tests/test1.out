warning: type cast int to float, line 5
type error: int expression expected, line 6
